pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1949,5 +1949,7 @@ EOT
     testTarget = "libarchive-test --test-options='-j1'";
   };
 
-  reactive-banana = super.reactive-banana.override { hashable = self.hashable_1_2_7_0; semigroups = self.semigroups_0_18_5;};
+  # unrestrict bounds for hashable and semigroups
+  # https://github.com/NixOS/nixpkgs/pull/125893
```

Comment:
Can you instead open an issue in <https://github.com/HeinrichApfelmus/reactive-banana> saying that their bounds are to strict and that they should update them on hackage?